server/studyspaces.py

```diff
@@ -0,0 +1,90 @@
+from flask import request, jsonify
+from .base import *
+from server import app
+from bs4 import BeautifulSoup
+import urllib2
+import requests
+
+def getIDs(): # extracts the ID's of the room into groupStudyCodes
```

Comment:
Use pythonic variable names `get_ids()` and use docstrings `"""This is a function docstring"""`
See [PEP 8](https://www.python.org/dev/peps/pep-0008/) for a detailed Python styleguide. While developing, you should always use flake8, pylint, etc to make sure your code is elegant/clean.